FEAT:  화장실 정보 구현 완료(#6)

kick-sim opened the pull request:
### 🔖작업내용

1. 테스트데이터를 DB에 저장하기 (화장실 데이터를 받아서 저장하는 방식으로 구현)
2. 위도, 경도를 입력 받아서 해당 위치 주변의 무료 화장실 리스트 반환
3. 위도, 경도를 입력 받아서 해당 위치 주변의 유료 화장실 리스트 반환
4. 화장실 키 받아서 화장실 상세 정보 반환
5. swagger 설정

### 📨관련 이슈

Resolved: #6

kick-sim committed:
최신화

kick-sim committed:
최신화

kick-sim committed:
Merge remote-tracking branch 'origin/develop' into develop

kick-sim committed:
Merge remote-tracking branch 'upstream/develop' into develop

kick-sim committed:
Feat #6 화장실 정보

위도, 경도를 입력 받아서 해당 위치 주변의 유/무료 화장실 리스트 반환
테스트데이터를 DB에 저장하기
화장실 키 받아서 화장실 상세 정보 반환

Fix: #6

kick-sim committed:
Merge remote-tracking branch 'upstream/develop' into develop

# Conflicts:
#	src/main/java/com/project/chamjimayo/controller/dto/ErrorCode.java

kick-sim committed:
gitingnore 다시 적용

kick-sim committed:
Feat #6 화장실 정보 swagger 설정, 반환값 일부 수정

swagger 설정, 공공화장실/유료화장실 데이터 입력시 화장실 Id와 화장실 이름이 반환되도록 변경

Resolves: #6

kick-sim committed:
DEL #6 RestroomExceptionHandler 삭제

try catch로 예외처리해서 필요없어서 삭제

Resolves: #6

kick-sim committed:
FIX #6 swagger 수정

swagger 에러 메시지 수정

Resolves: #6

kick-sim committed:
FIX #6 주변 화장실 검색 수정

주변 화장실 검색 로직 수정

Resolves: #6

kick-sim committed:
Merge remote-tracking branch 'upstream/develop' into develop

# Conflicts:
#	src/main/java/com/project/chamjimayo/controller/dto/ApiStandardResponse.java
#	src/main/java/com/project/chamjimayo/controller/dto/ErrorCode.java
#	src/main/java/com/project/chamjimayo/domain/entity/Restroom.java
#	src/main/resources/config

kick-sim committed:
Feat #6 exceptionHandler처리

exceptionHandler 처리 및 예외 처리 로직 변경

Resolves: #6

hyuntae99 commented on src/main/java/com/project/chamjimayo/controller/RestroomController.java:
[SwaggerConfig](https://github.com/chamjimayo/backend/blob/develop/src/main/java/com/project/chamjimayo/controller/config/SwaggerConfig.java)에서 "Api 키 오류", "Method Not Allowed", "Internal Server Error" 자동으로 추가해줘서
위에 해당하는 오류는 @ApiResponse 안 해줘도 돼
그리고 example 형식이 바뀌었더라고?? 나도 [이거](https://github.com/chamjimayo/backend/blob/develop/src/main/java/com/project/chamjimayo/controller/AuthController.java) 따라해서 통일하는게 좋을 듯?

kick-sim committed:
Fix #6 swagger 설정 변경, RestroomDetail 반환타입 변경, 엔티티 수정

@ApiResponse 중복되는 부분 삭제, swagger example 형식 변경, Restroom RestroomDetail 반환타입 변경, restroom 엔티티에 필요없는 코드 삭제, user 엔티티에 사용중인 화장실, 사용한 화장실 리스트 필드 추가

Resolves: #6

kick-sim committed:
Fix #6 근처 화장실 반환값 수정

review, equipment, restroommanager 반환값 id와 이름만 반환하도록 수정

Resolves: #6

seokho-1116 reviewed (approved):
수고 많았으

hyuntae99 reviewed (approved):
수고했어!
